tests/general/InvalidLinkBearTest.py

```diff
@@ -154,10 +153,7 @@ def test_run(self):
         http://httpbin.org/status/404
         http://httpbin.org/status/410
         http://httpbin.org/status/500
-        http://httpbin.org/status/503
-        http://www.notexample.com/404
-        http://exampe.com/404
-        http://example.co.in/404"""
+        http://httpbin.org/status/503"""
```

Comment:
put the trailing `"""` on a new line, so that it doesnt need to move around and the line "http://httpbin.org/status/503" doesnt need to be modified by your change.